### ISSUE TITLE
Can't remove gnome-shell theme

### ISSUE THREAD

**Issue by fir3-1ce** [vinceliuice/Graphite-gtk-theme#110]
I'm stuck with this theme, i cannot completely remove it. I originally used the `-l` option when I installed. I tried going back and using the `-u` option. I don't know if I'm supposed to use `sudo` or not, I don't want to mess things up more than they already are. The GDM screen appears to be permanent too. I don't have any extra files in my `~/.config/gtk-4.0` folder either.

I'm on Ubuntu 22.04 LTS with GNOME 42.9

**Comment by vinceliuice** [comment]
`sudo ./install.sh -g -u`

**Comment by fir3-1ce** [comment]
Thanks, I ended up using `sudo ./install.sh -g -u` and `sudo ./install.sh -l -u` to be safe and it worked. I don't know how the symlinking to libadwaita works, but I was having problems with the theme where the gnome-shell wasn't scaled properly to the rest of my display. I use a 4K monitor and I have the "big text" accessibility option checked. 

Are the shell elements unfinished, or this another error on my system? I tried some other themes just for the shell and they seemed to have the properly scaled text. This made me think that it was maybe something with the `-l` flag in your install script. For now I am using Yaru-dark for the shell and I actually went ahead and reinstalled Graphite for applications.
